apps/dh/api-dh/project.json

```diff
@@ -1,4 +1,5 @@
 {
+  "$schema": "..\\..\\..\\node_modules\\nx\\schemas\\project-schema.json",
```

Comment:
Suggestion: 
```suggestion
  "$schema": "node_modules/nx/schemas/project-schema.json",
```

And this one goes for all the nx project-schema changes :)

Reply:
Just pushed a commit with changes. The `../../../` part is still needed though :)

Reply:
I just checked, and ./node_modules does not work. Resolving this one :)

Reply:
Because the lookup starts from where the file is placed. Those on root start with `./node_modules/...`.